processing/src/main/java/io/druid/segment/ColumnSelectorBitmapIndexSelector.java

```diff
@@ -110,6 +110,21 @@ public void inspectRuntimeShape(RuntimeShapeInspector inspector)
   }
 
   @Override
+  public boolean hasMultipleValues(final String dimension)
+  {
+    if (isFilterableVirtualColumn(dimension)) {
```

Comment:
Maybe this logic isn't really right though. The idea is that a column name should be treated as a virtual column if it defined as one, otherwise it should be treated as a regular column. So it should probably be `isVirtualColumn` rather than `isFilterableVirtualColumn`. I'll change all methods, including this one.